CONTRIBUTING.md

```diff
@@ -1,65 +1,251 @@
+
+1. When completing a review,
+   favor [approving the PR][approve-pr] rather than requesting changes,
+   whenever possible.
+
+   1. If all of your comments were suggestions rather than issues,
+      approve the PR and trust the submitter to read and consider them
+      before merging.
+      Click **Approve** and change the label from
+      [**review.tech**][review-tech-label] to
+      [**review.copy**][review-copy-label].
+
+   1. If the PR needs concrete changes before you can approve,
+      note the reason why,
+      click **Comment** and change the label from
+      [**review.tech**][review-tech-label] to
+      [**review.await-update**][review-await-label].
+
+### Perform a content review
+
+In general, follow these practices.
+
+* Favor trusting the writer.
+  If they stand by the changes and the copy issues are minimal,
+  _approve the PR_.
+* Provide suggested changes with corrected copy to make your change clear.
+
+To conduct a consistent copy review in the Flutter website:
+
+1. [Review a PR][request-pr] when it meets the following conditions:
+
+   1. The writer [requested your review][request-review] of the PR.
+
+   1. The writer applied the [**review.copy**][review-copy-label]
+      label to the PR.
+
+   1. The writer didn't set the PR to [Draft mode][set-draft].
+      You can provide feedback to a draft issue,
+      but you should understand that the author's still working on the draft.
+
+1. Check that text and code examples are consistent with one another.
+
+1. When adding comments,
+   be clear if you're making a suggestion or pointing out something that
+   definitely needs to change.
+
+1. Stay consistent with the
+   [Google developer documentation style guide][style-guide] (GDDSG).
+
+1. When completing a review,
+   prefer approving a PR rather than requesting changes, whenever possible.
+
+   1. If all of your comments were suggestions rather than issues,
+      just approve and trust the submitter to read and consider them before merging.
+      Click **Approve** and change the label
+      from [**review.copy**][review-copy-label] to **st.RFM.%**.
+
+   1. If there are concrete changes needed before you can approve,
+      note the reason why, click **Comment** and change the label
+      from **review.copy** to **review.await-update**.
+
+## Resources for contributors and reviewers
+
+* Semantic Line Breaks: https://sembr.org
+* Google Developer Documentation Style Guide: https://developers.google.com/style
+* Chicago Manual of Style: https://www.chicagomanualofstyle.org
+
+[approve-pr](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews)
+[add-comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request)
+[search-issue](https://www.freecodecamp.org/news/github-search-tips/)
```

Comment:
We should reference official docs as much as possible: https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests